src/core/net/ip6.cpp

```diff
@@ -1186,29 +1189,34 @@ otError Ip6::HandleDatagram(Message &aMessage, Netif *aNetif, const void *aLinkM
     }
     else
     {
+        // unicast
```

Comment:
```suggestion
```

Reply:
the comment tells that this else block is for unicast. Doesn't it make the code more readable?